backend/controllers/appointments/confirmAppointment.js

```diff
@@ -175,14 +175,10 @@ const confirmAppointmentFactory = ({ elite2Api, appointmentsService, logError })
           numberAdded: details.numberOfAppointments,
           lastAppointment: details.endDateShortFormat,
         }
-        const formattedName = sanitizer.sanitize(formatName(firstName, lastName))
-
-        const prisonerName =
-          formattedName && formattedName[formattedName.length - 1] !== 's' ? [formattedName, 's'] : [formattedName]
 
         res.render('confirmAppointments.njk', {
-          titleHtml: `${prisonerName[0]}&rsquo;${prisonerName[1] || ''} appointment has been added`,
           addAppointmentsLink: `/offenders/${offenderNo}/add-appointment`,
+          prisonerName: sanitizer.sanitize(formatName(firstName, lastName)),
```

Comment:
Not familar with this @JamieDraperUK. Any reason we do this `sanitizer` here?

Reply:
It's to stop XSS attacks given that the name presented in the view was unsafe. An attacker could change an offenders name to `<script> alert('do something malicious') </script> Bob Doe` Not sure if there is an easy way to solve it. I tried using | safe but because the older code did some string building in the view it didn't work very well.